src/driver/express/ExpressDriver.ts

```diff
@@ -114,21 +115,24 @@ export class ExpressDriver extends BaseDriver implements Driver {
 
                 const action: Action = {request, response, next};
                 const checkResult = this.authorizationChecker(action, actionMetadata.authorizedRoles);
-                if (isPromiseLike(checkResult)) {
-                    checkResult.then(result => {
-                        if (!result) {
-                            return this.handleError(new AccessDeniedError(action), actionMetadata, action);
 
-                        } else {
-                            next();
-                        }
-                    });
-                } else {
-                    if (!checkResult) {
-                        this.handleError(new AccessDeniedError(action), actionMetadata, action);
+                const handleError = (result: any) => {
+                    if (!result) {
+                        return this.handleError((
```

Comment:
multiline params this way do not look good, simplify this. extract error class in a separate variable if needed